City-of-Helsinki/helsinki-paatokset
# PP-149: Platform and subtheme updates, subtheme cleanup

**To test**
- Checkout branch
- Run make fresh (if you have created a database dump) or make new
- Check that no database updates are pending with make drush-updb
- Check that there are no pending updates: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
